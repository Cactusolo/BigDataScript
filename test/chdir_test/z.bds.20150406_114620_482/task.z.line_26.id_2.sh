#!/bin/bash -e

cat chdir_test_file.txt