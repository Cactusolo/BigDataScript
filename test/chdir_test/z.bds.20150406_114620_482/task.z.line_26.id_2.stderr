cat: chdir_test_file.txt: No such file or directory
